packages/stream-metadata/src/opensea.ts

```diff
@@ -5,16 +5,20 @@ import { FastifyBaseLogger } from 'fastify'
 import { config } from './environment'
 import { spaceDapp } from './contract-utils'
 
-const getOpenSeaAPIUrl = (space: SpaceInfo) => {
+const getOpenSeaAPIUrl = (logger: FastifyBaseLogger, space: SpaceInfo) => {
 	const spaceOwnerAddress = config.web3Config.base.addresses.spaceOwner
+	const chainId = config.web3Config.base.chainId
 	const tokenId = BigNumber.from(space.tokenId).toString()
 
-	if (space.networkId === String(BASE_MAINNET)) {
+	if (chainId === BASE_MAINNET) {
 		return `https://api.opensea.io/api/v2/chain/base/contract/${spaceOwnerAddress}/nfts/${tokenId}/refresh`
-	} else if (space.networkId === String(BASE_SEPOLIA)) {
+	} else if (chainId === BASE_SEPOLIA) {
+		return `https://testnets-api.opensea.io/api/v2/chain/base_sepolia/contract/${spaceOwnerAddress}/nfts/${tokenId}/refresh`
+	} else if (chainId === 31337) {
 		return `https://testnets-api.opensea.io/api/v2/chain/base_sepolia/contract/${spaceOwnerAddress}/nfts/${tokenId}/refresh`
```

Comment:
should localhost just return example.com or something simple?

Reply:
This endpoint is a weird spot for local development, I would skip for now